src/tasks/task.ts

```diff
@@ -0,0 +1,86 @@
+import assign from "../util/assign";
+import { generateId } from "../util/unique-id";
+import { beacon } from "../util/beacon";
+import { getClientInfo } from "../util/client-info";
+
+interface State {
+  hasRan: boolean;
+}
+
+class Task {
+  public config: any = {};
```

Comment:
This doesn't need to be `any` as we will no this structure ahead of time.

Reply:
yes, I need to clean up a lot of the `any` typings. thanks for the reminder.